plugins/terminal-proxy/lib/terminalProxy.js

```diff
@@ -408,8 +408,8 @@ const WEBSOCKET_REASON_TERMPROXY_GOING_AWAY = 4000;
 TerminalWebsocketProxy.prototype.handleData = function(data, ws) {
   var t = this;
   try {
-    t.logger.debug(t.identifierString()+' Received host data. Length='+data.length);
-    t.logger.log(t.logger.FINER,t.identifierString()+' Content of host data=\n'+data);
+    t.logger.debug("ZWED0277I", t.identifierString(), data.length); //t.logger.debug(t.identifierString()+' Received host data. Length='+data.length);
+    t.logger.debug("ZWED0138I", t.identifierString(), data); //t.logger.log(t.logger.FINER,t.identifierString()+' Content of host data=\n'+data);
```

Comment:
Same concern as above... you really do not want to print this by accident, it can be a lot of stuff. best to use logger.trace

Reply:
Corrected to trace.